composer.json

```diff
@@ -15,14 +15,13 @@
         "composer/installers": "~1.5",
         "drupal-composer/drupal-scaffold": "^2.5.2",
         "drupal/config_devel": "~1.2",
-        "drupal/console": "~1.0",
         "drupal/devel": "~1.2",
         "drupal/drupal-extension": "~4.0",
         "drush/drush": "~9.0@stable",
         "openeuropa/behat-transformation-context" : "~0.1",
         "openeuropa/code-review": "~1.0@beta",
         "openeuropa/drupal-core-require-dev": "^8.7",
-        "openeuropa/task-runner": "~1.0@beta",
+        "openeuropa/task-runner": "^1.0.0-beta6",
```

Comment:
Why change `~` into `^`?

Reply:
Changed. Looks like it is better `~`. Initially used the same declaration of constraints as in https://github.com/ec-europa/ewcms/blob/1.5.0/composer.json#L46